pythonsd/templates/pythonsd/organizers.html

```diff
@@ -5,39 +5,40 @@
 
 
 {% block main %}
-<div class="container mt-3">
+<div class="container mt-4">
 <h1>Organizers</h1>
 
 <p>If you would like to reach out, please contact the <a href="mailto:sandiegopython-organizers@googlegroups.com">Python SD Organizers</a>.</p>
 
 
 {% if organizers %}
-<div class="row row-cols-2 row-cols-md-4">
+<div class="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-4 gap-12 mb-16 ">
```

Comment:
```suggestion
<div class="grid grid-cols-1 sm:grid-cols-2  md:grid-cols-3 lg:grid-cols-4 gap-12 mb-16 ">
```
640-767px isn't the most common viewport width, but this will remove unnecessary white space for users with that resolution.